google/cloud/pubsub/publisher_connection.h

```diff
@@ -125,6 +125,13 @@ std::shared_ptr<pubsub::PublisherConnection> MakePublisherConnection(
     std::unique_ptr<pubsub::RetryPolicy const> retry_policy,
     std::unique_ptr<pubsub::BackoffPolicy const> backoff_policy);
 
+std::shared_ptr<pubsub::PublisherConnection> MakePublisherConnection(
+    pubsub::Topic topic, pubsub::PublisherOptions options,
+    pubsub::ConnectionOptions connection_options,
+    std::vector<std::shared_ptr<PublisherStub>> stubs,
```

Comment:
Is throwing at the `at()` in `Child()` the intended response to an empty vector?  If so, perhaps add a comment that it should not be empty.  If not, it seems like `MakePublisherConnection()` should probably return `nullptr` in that circumstance.

Reply:
I am not sure `nullptr` is great, the result goes into the constructor of `Publisher` and `TopicAdminClient`.  On the other hand, largely it does not matter because this is in `internal::` and all the public interfaces create non-empty stub lists. Let me know what you think.

Reply:
Sorry ... I missed that this was inside `pubsub_internal` despite being in a public header.